libminifi/test/rocksdb-tests/DBContentRepositoryTests.cpp

```diff
@@ -179,7 +179,7 @@ TEST_CASE("Delete NonExistent Claim", "[TestDBCR4]") {
 
   std::string readstr;
 
-  // -1 tell us we have an invalid stream
+  // error tells us we have an invalid stream
```

Comment:
it would be better to remove this comment, as we don't check for error here (also at line 230)

Reply:
done in 141917b